tests/aws/services/events/test_events_targets.py

```diff
@@ -37,6 +37,116 @@
 #   - Container (pro)
 #   - Redshift (pro)
 #   - Sagemaker (pro)
+class TestEventsTargetCloudWatchLogs:
+    @markers.aws.validated
+    def test_put_events_with_target_cloudwatch_logs(
+        self,
+        events_create_event_bus,
+        events_put_rule,
+        aws_client,
+        snapshot,
+        account_id,
+        region_name,
+    ):
+        snapshot.add_transformers_list(
+            [
+                snapshot.transform.key_value("EventId"),
+                snapshot.transform.key_value("RuleArn"),
+                snapshot.transform.key_value("EventBusArn"),
+            ]
+        )
+
+        # Step 1: Create an EventBridge event bus
```

Comment:
Appreciate the comments here! Think a lot of the code is self-documenting though i.e `events_create_event_bus` signals the bus is being created.

For future reference, don't think you need to worry about adding comments/docs each time though (unless there's something particularly weedy that you think justifies the extra info 🧠).

If you think the extra docs are necessary, perhaps rather add a docstring to the test function that explains what the test does etc.